single-line-diagram-viewer/src/main/java/com/powsybl/sld/viewer/SingleLineDiagramViewer.java

```diff
@@ -581,7 +631,11 @@ private void createParametersPane() {
         int rowIndex = 0;
 
         Button fitToContent = new Button("Fit to content");
-        fitToContent.setOnAction(event -> {
+        // FIXME : not implemented
+        fitToContent.setDisable(true);
```

Comment:
I think you can deduce the needed zoom from the size of the webview (in screen pixels) and the size of the svg, which you can get from its width/height parameters. Too bad they're not in the metadata. But parsing the svg string is ok at first.